### DISCUSSION TITLE
when clause not working as expected for resourceDirname

### DISCUSSION THREAD

**Discussion by amitjoshi438** [microsoft/vscode-discussions#311] [Extension Development QnA] [answered]
I want to show an explorer menu item when the folder matches a certain directory.
`"when": "resourceDirname == web-pages" `
but this is not working, and menu item is not showing up even for web-pages folder.

Is it a bug or should I do it differently?

TIA

**Comment by isidorn** [comment, marked as answer]
This should work. If it does not I suggest you create a new issue with steps in https://github.com/microsoft/vscode/issues

fyi @lramos15
